Harvest.Web/Controllers/Api/TicketController.cs

```diff
@@ -23,6 +23,21 @@ public TicketController(AppDbContext dbContext, IUserService userService)
             this._userService = userService;
         }
 
+        // create a new ticket via react
+        [HttpGet]
+        [Authorize(Policy = AccessCodes.PrincipalInvestigator)]
+        public async Task<ActionResult> ListTickets(int projectId, bool topOnly = true)

```

Comment:
you don't need to put `Tickets` in the name if it's already the ticket controller.  `/ticket/list/` i think is enough info on what is happening